opencontrail/files/3.0/config.global.js

```diff
@@ -179,7 +183,11 @@ config.files.download_path = '/tmp';
 /* WebUI Redis Server */
 config.redis_server_port = '6379';
 config.redis_server_ip = '{{ web.cache.host }}';
+{%- if common.vendor == "juniper" %}
+config.redis_dump_file = '/var/lib/redis/dump.rdb';
```

Comment:
it's the location where it's created by package if I do understand.  We were forced to fix the path in order to make it runing. is the redis dump file created by formula, I don't think so..